File: app/src/main/java/org/ionchain/wallet/callback/OnLoadingView.java
package org.ionchain.wallet.callback;

/**
 * author  binny
 * date 5/8
 * <p>
 * <p>
 * 正在加载
 */
public interface OnLoadingView {
    /**
     * 正在加载
     */
    void onLoadStart();

    /**
     * 加载完成
     */
    void onLoadFinish();
}

File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnLoadingView.kt
package org.ionchain.wallet.mvp.callback

/**
 * author  binny
 * date 5/8
 *
 *
 * 正在加载
 */
interface OnLoadingView {
    /**
     * 正在加载
     */
    fun onLoadStart()

    /**
     * 加载完成
     */
    fun onLoadFinish()
}
